### PR TITLE
feat:add a 'namespace' parameter to the data processing API interface.

### PR DESCRIPTION
**Pull request by wangxinbiao**
add a 'namespace' parameter to the data processing API interface for graphql.

### CODE REVIEW

File: graphql-server/go-server/graph/schema/dataprocessing.graphqls [hunk 1]
```diff
@@ -41,6 +43,8 @@ input AddDataProcessInput {
   data_process_config_info: [DataProcessConfigItem!]
   bucket_name: String!
   version_data_set_name: String!
+  namespace: String!
+  create_user: String!
```
Comment (bjwswang) [inline graphql-server/go-server/graph/schema/dataprocessing.graphqls:47]: @wangxinbiao do you mean `creator` ?

File: graphql-server/go-server/graph/schema/dataprocessing.graphqls [hunk 2]
```diff
@@ -41,6 +43,8 @@ input AddDataProcessInput {
   data_process_config_info: [DataProcessConfigItem!]
   bucket_name: String!
   version_data_set_name: String!
+  namespace: String!
```
Comment (bjwswang) [inline graphql-server/go-server/graph/schema/dataprocessing.graphqls:46]: @wangxinbiao  I don't see anywhere that. you use this `namespace` and `creator`.

File: graphql-server/go-server/graph/schema/dataprocessing.graphqls (hunk 1 above)
Comment (wangxinbiao) [reply graphql-server/go-server/graph/schema/dataprocessing.graphqls:47]: yes

File: graphql-server/go-server/graph/schema/dataprocessing.graphqls (hunk 2 above)
Comment (wangxinbiao) [reply graphql-server/go-server/graph/schema/dataprocessing.graphqls:46]: python code hasn't been submitted.

File: graphql-server/go-server/graph/schema/dataprocessing.graphqls (hunk 1 above)
Comment (nkwangleiGIT) [reply graphql-server/go-server/graph/schema/dataprocessing.graphqls:47]: so, update the column name to creator

**Comment by bjwswang** [comment]
closed since we merged this commit in dev branch https://github.com/bjwswang/arcadia/commits/gql/

**Pull request closed without merging**